gen/gen.go

```diff
@@ -124,8 +124,22 @@ func Gen(exercise string, j interface{}, t *template.Template) error {
 		}
 	}
 
+	// read tests.toml file to find which test cases should be excluded
+	tomlFile := filepath.Join(dirExercise, ".meta", "tests.toml")
+	log.Printf("[LOCAL] reading tests.toml file from exercise directory %s\n", tomlFile)
+	excludedTests, err := getExcludedTestCases(tomlFile)
+	if err != nil {
+		return fmt.Errorf("[LOCAL] unable to read tests.toml file : %v", err)
+	}
+
+	// remove the excluded test cases from the source json
+	filteredJSrc, err := filterTestsJson(jSrc, excludedTests)
+	if err != nil {
+		return fmt.Errorf("Unable to filter tests cases : %v", err)
```

Comment:
Error messages should not be capitalized: https://stackoverflow.com/questions/68792696/why-error-messages-shouldnt-end-with-a-punctuation-mark-in-go#:~:text=Error%20strings%20should%20not%20be%20capitalized%20(unless%20beginning%20with%20proper,usually%20printed%20following%20other%20context.